### PR TITLE
add-faker-integration: removed 'name' because it is an existing faker option

### PR DESCRIPTION
**Pull request by franziskahahn**
Hey guys,
I saw that "name" is already (is it new, I never saw it before?! :D) an "official" faker formatter.

So I deleted the self-written formatter. 
I decided to keep the configuration skeleton because I am sure we will need individual stuff in the further development of slimdump.

### CODE REVIEW

**Comment by franziskahahn** [comment]
I think the decreased coverage check is not correct..

File: src/Webfactory/Slimdump/Config/FakerReplacer.php [hunk 1]
```diff
@@ -20,11 +20,12 @@ class FakerReplacer
     private $faker;
 
     /**
-     * configuration for method which are not part of faker itself
+     * configuration for method which are not part of faker itself.
+     * You can implement a new method "methodName" which returns for example a
+     * format: 'suffix' => 'methodName',
      * @var array
      */
     private $replacementOptions = [
```
Comment (mpdude) [inline src/Webfactory/Slimdump/Config/FakerReplacer.php]: Can't we remove this as well?

**Comment by mpdude** [comment]
Sorry @franziskahahn, I somehow lost sight of this.

I only quickly skimmed `FakerReplacer` on my mobile phone, so maybe I am wrong. But, if we remove this special case handling, can't we get rid of even more code there? I mean, we need not keep hooks and extension points around for things we don't actually use (yet).

As we haven't had a release since your initial contribution, I see no Problem in removing stuff.

**Comment by franziskahahn** [comment]
Hey @mpdude 
I also lost sight of this. I didn't deleted it yet because I thought I can find a new option but now I think that faker already provides all necessary options. I will delete the stuff.

**Comment by franziskahahn** [comment]
I removed the config array

**Comment by mpdude** [comment]
Danke @franziskahahn!

**Pull request merged**